docs/contributor/01-11-ci-cd.md

```diff
@@ -0,0 +1,30 @@
+# CI/CD
+
+## Pipelines running on pull requests
+
+The following CI jobs are part of the development cycle. They verify the functional correctness of Application Connector Manager but do not verify the contract concerning Kyma's Lifecycle Manager.
+
+| Name                                                                                                                                                  | Required | Platform   | Description                                                                                                                                                                                             |
+|-------------------------------------------------------------------------------------------------------------------------------------------------------|----------|------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| [`pre-application-connector-manager-build`](https://github.com/kyma-project/test-infra/blob/main/templates/data/generic_module_data.yaml#L1059)       | true     | prow       | Builds Application Connector Manager's image and pushes it to the `dev` registry.                                                                                                                       |
+| [`pre-application-connector-manager-presubmit-scanner`](https://github.com/kyma-project/test-infra/prow/jobs/test-infra/presubmit-scanner.yaml#L556)  | true     | prow       | Run Gitleaks presubmit scanner to detect committed sensitive data for Application Connector Manager.                                                                                                    |
```

Comment:
This link is broken. Maybe it should be this one - https://github.com/kyma-project/test-infra/blob/main/prow/jobs/test-infra/presubmit-scanner.yaml ?